multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -73,8 +73,13 @@ type ClusterConditionType string
 const (
 	// ClusterReady indicates whether Cluster is ready and connected.
 	ClusterReady ClusterConditionType = "Ready"
-	// ClusterIsLeader indicates whether Cluster is leader.
-	ClusterIsLeader ClusterConditionType = "IsLeader"
+	// ClusterIsElectedLeader indicates whether the leader cluster is the elected leader.
+	// Used on Member clusters only.
+	ClusterIsElectedLeader ClusterConditionType = "IsElectedLeader"
+	// ClusterWatchesLocalResources indicates whether the member cluster has elected the
+	// local cluster as the elected leader and watches resources from it.
+	// Used on in Leader clusters.
```

Comment:
Remove "in".

Reply:
Sure, I fixed it, maybe in reads better than on